docs/product/guidelines/accessibility.html

```diff
@@ -0,0 +1,359 @@
+
+<section class="stacks-section">
+{% stacks-markdown %}
+## Target
+
+All Stack Overflow product UIs must conform to the AA conformance level of the Web Content Accessibility Guidelines (WCAG) 2.2 with a few exceptions around color contrast documented below.
+
+### High contrast modes
+
+When high contrast mode is enabled, Stack Overflow product UIs must meet or exceed the [Success Criterion 1.4.3 Contrast (Enhanced)](https://www.w3.org/TR/WCAG22/#contrast-enhanced) of the [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/) and should conform to the remaining [AAA conformance level rules](https://www.w3.org/TR/WCAG22/#cc1) when reasonably achievable. This only applies to the subset of Stack Overflow products that provide high contrast modes. *Note: not all Stack Overflow products are expected to support high contrast modes*.
+
+{% endstacks-markdown %}
+
+</section>
+
+<section class="stacks-section">
+{% stacks-markdown %}
+## Visual accessibility
+
+Stack Overflow product UIs MUST conform to a custom conformance level of the Accessible Perceptual Contrast Algorithm (APCA). This custom conformance level replaces the [AA conformance level](https://www.w3.org/TR/WCAG22/#cc1) of the [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/) for color contrast.
+
+{% endstacks-markdown %}
+
+{% stacks-markdown %}
+### Color & contrast
+
+Stacks aims to equip you with an accessible color palette and has been tested against WCAG AA, WCAG AAA and the newer APCA color standards. Most of our color combinations meet WCAG AA and APCA levels defined below. We also offer high contrast mode which offers a greater level of contrast. 
+{% endstacks-markdown %}
+
+{% stacks-markdown %}
+#### Contrast ratios
+
+Contrast is the difference in luminance or color between any two elements. All visual readers require ample luminance contrast for fluent readability. Stack Overflow products must conform to a custom conformance level of the Accessible Perceptual Contrast Algorithm (APCA). Based on our custom conformance level, all text must have a Lightness contrast (Lc) value of 60, body copy must have a Lc value of 75, icons must have a Lc value of 45, and placeholder and disabled text must have a Lc of 30. These numbers will be negative when calculating for dark mode.
+{% endstacks-markdown %}
+
+    {% capture correctElement %}
+        <button class="s-btn s-btn__filled" role="presentation">Button</button>
+        <div class="bar-pill ta-center px8 py2 bg-green-400 fc-white">Robotics</div>
+    {% endcapture %}
+
+    {% capture incorrectElement %}
+        <button class="s-btn s-btn__filled bg-theme-secondary-200" role="presentation">Button</button>
+        <div class="bar-pill ta-center px8 py2 bg-green-400 fc-green-300">Robotics</div>
+    {% endcapture %}
+
+    {% render 'accessibility/comparison.html',
```

Comment:
**Sidenote (No action required)**

I look at something like this and I cannot stop but thinking how much more elegant/readable would be to use a Svelte component. This gives me the Razor vibe 😀

Reply:
I absolutely agree. I would love to rearchitect our docs to use SvelteKit for reasons like that one (and many more)